README.rst

```diff
@@ -421,6 +421,29 @@ Usage example:
                           create(Builder('folder')).created())
 
 
+It is convenient to install the ticking creator globally so that when a builder
```

Comment:
`...so that when...` sounds a little strange to me.
Probably better `It is convenient to install the ticking creator globally so if a builder creates other builders, it also ticks the clock.`